### PR TITLE
Update alt text for binary file image

### PR DESCRIPTION
**Pull request by JLLeitschuh**
For screen readers and the like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ If any are found that do not match the SHA-256 checksums of our official release
 Simply add this action to your workflow **after** having checked out your source tree and **before** running any Gradle build:  
 
 ```yaml
-uses: gradle/wrapper-validation-action@v1
+uses: gradle/wrapper-validation-action@releases/v1
```
Comment (eskatos) [inline README.md]: :x: why change this to point to a branch instead of a release tag ?

File: README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/wrapper-validation-action@releases/v1
```
Comment (eskatos) [inline README.md]: :x: why change this to point to a branch ?

File: README.md (hunk 1 above)
Comment (JLLeitschuh) [reply README.md]: That was unintentional and not part of the change I meant to make.

File: README.md (hunk 2 above)
Comment (JLLeitschuh) [reply README.md]: That was unintentional and not part of the change I meant to make.

**Comment by JLLeitschuh** [comment]
Resolved the issues.

**Pull request merged**